src/userAccount/views/OrderDetails/Page.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+import { AddressSummary, CartTable, NotFound } from "../../../components";
+import { LineI } from "../../../components/CartTable/ProductRow";
+import { priceToString } from "../../../core/utils";
+import { OrderById_order, OrderById_order_lines } from "./types/OrderById";
+import {
+  OrderByToken_orderByToken,
+  OrderByToken_orderByToken_lines
+} from "./types/OrderByToken";
+
+import { orderHistoryUrl } from "../../routes";
+
+const extractOrderLines = (
+  lines: Array<OrderById_order_lines | OrderByToken_orderByToken_lines>
+): LineI[] => {
+  return lines
+    .map(line => ({
+      quantity: line.quantity,
+      totalPrice: priceToString({
+        amount: line.quantity * line.variant.price.amount,
+        currency: line.variant.price.currency
+      }),
+      ...line.variant,
+      name: line.productName
+    }))
+    .sort((a, b) => b.id.toLowerCase().localeCompare(a.id.toLowerCase()));
+};
+
+const Page: React.FC<{
+  guest: boolean;
+  order: OrderById_order | OrderByToken_orderByToken;
+}> = ({ guest, order }) =>
+  order ? (
+    <>
+      {!guest && (
+        <Link className="order-details__link" to={orderHistoryUrl}>
+          Go back to Order History
+        </Link>
+      )}
+      <h3>Your order nr: {order.number}</h3>
```

Comment:
It looks weird to me. The format `Order #{order_number}` wouldn't be better looking?

Reply:
I agree.